frontend/src/components/dashboard/subdomain/SubdomainInfoTooltip.tsx

```diff
@@ -35,6 +36,9 @@ export const SubdomainInfoTooltip = () => {
 
 const MobileExplainerModal = () => {
   const modalState = useOverlayTriggerState({});
+  const profileData = useProfiles();
```

Comment:
It doesn't matter that much since `useProfiles` deduplicates requests (i.e. including it in multiple components doesn't result in multiple requests to the profiles API), but it can get somewhat annoying in unit tests where we'd have to mock the return value of `useProfiles` once for every component that calls it. Thus, you could also consider passing it in as a Prop - AFAIK the `<SubdomainInfoTooltip>` is included in the profile page, which should already have the profile data available.

Reply:
Great idea, glad to get a second pair of eyes on this.